Connected/ProjectSettings/ProjectSettings.asset

```diff
@@ -691,7 +691,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 0
+  activeInputHandler: 1
```

Comment:
Which one should we be using, this is driving me insane haha

Reply:
What even is this setting? And how did I manage to change it?

Reply:
You probably clicked yes when it prompted on Unity startup